src/BBT.StrategyPattern/Tools/GenericInstanceCreator.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace BBT.StrategyPattern.Tools
```

Comment:
Change namespace to `BBT.StrategyPattern`. For the consumer it's much more convenient to have all classes in a single namespace instead of requiring to import 3 namespaces.